src/hooks/useSnackbar.ts

```
@@ -0,0 +1,88 @@
+     */
+    const showWarning = (message: string, autoHideDuration: number = 6000) => {
+        dispatch(showWarningSnackbar({ message, autoHideDuration }));
+    };
+
+    /**
+     * Displays an info snackbar notification with the given message.
+     * @param message The message to display in the snackbar.
+     * @param autoHideDuration The duration in milliseconds for which the snackbar should stay open.
+     *     If not provided, defaults to 6000ms.
+     */
+    const showInfo = (message: string, autoHideDuration: number = 6000) => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)